dndnode_ass/index.ts

```diff
@@ -0,0 +1,30 @@
+import * as Races from "./races";
+
+const express = require('express')
+const cors = require('cors')
+const app = express()
+const port = 3030
+
+app.use(cors());
+
+
+app.get('/Races', (req: any, res: any) => {
+
+    let raceNames: string[] = [];
+
+    for (let race of Races.allRaces) {
+        raceNames.push(race.raceName);
+    }
+
+    res.send(JSON.parse(JSON.stringify(raceNames)))
```

Comment:
Bruh stringify and parse lol what are you doing

Anyways this will nicely return js object as json + set the json headers:

```suggestion
    res.json(raceNames)
```